docs/rules.md

```diff
@@ -925,6 +925,12 @@ This rule is to check if the schemas used by private endpoint conform to the com
 
 Please refer to [private-endpoint-resource-schema-validation.md](./private-endpoint-resource-schema-validation.md) for details.
 
+### PropertiesTypeObjectNoDefinition
```

Comment:
Sometimes also called "Free-form Objects", if we want to add this bit in description.